sdk/loadtesting/azure-developer-loadtesting/swagger/README.md

```diff
@@ -11,6 +11,7 @@ security-scopes: https://cnt-prod.loadtesting.azure.com/.default
 artifact-id: azure-developer-loadtesting
 namespace: com.azure.developer.loadtesting
 generate-builder-per-client: true
+enable-sync-stack: true
```

Comment:
Include the autorest version used to generate this code.

Reply:
Added